### ISSUE TITLE
I'd love the Android App to be x86 enabled

### ISSUE THREAD

**Issue by AnneWielis** [martinstoeckli/SilentNotes#218]
I'm running WayDroid for all my Apps that work better on a big screen and a good keyboard. Unfortunately, SilentNotes won't install from the PlayStore, it says it's not compatible with my device - obviously because of the x86 processor.

Is there any chance that you might allow for the x86 architecture in your APKs?

**Comment by martinstoeckli** [comment]
Thanks for writing, I had a quick look at WayDroid and understood that it is an Android running on a Linux device? Could it be a solution to run the Windows version of SilentNotes then, maybe that's possible with WINE. I'm a bit reluctant to publish versions I cannot test myself.

**Comment by AnneWielis** [comment]
Unfortunately no, tried that already. I would be happy to assist in testing, though.

**Comment by martinstoeckli** [comment]
I have to take care that the required time for the distribution of a new version isn't increasing too much, at the moment it is about 2 hours just to build and package the new binaries, writing the release notes and uploading them to the different stores/homepage. But a solution would be to setup an automated build pipeline on GitHub.

So I will relate this issue to #215 , an automated building and publishing of the binaries would improve the situation and it would allow to publish different apks for different achitectures without increasing the size of the original apk. A small downside is that I would have to upload the private keys to the GitHub vault, but probably it's worth it.

**Comment by AnneWielis** [comment]
I see. Thanks for considering, anyway.

**Comment by martinstoeckli** [comment]
Currently I'm writing CI build-pipelines for the upcoming version 8, so building for different architectures could become possible. Is your issue still active and would `x64` be the correct platform, or is it indeed `x86` (for 32bit processors)?

**Comment by AnneWielis** [comment]
Yes, it is still relevant and yes, x64 is the right platform. I'd be happy to help testing.

**Comment by martinstoeckli** [comment]
Finally, version 8.0 is released and with this release I also automated the build process. On the release page you can find an apk for x64, but I'm really not sure if the configuration was correct for building this apk (and don't have such a device for testing), so I would be happy if you could make a test on your device.
  https://github.com/martinstoeckli/SilentNotes/releases/tag/v8.00.00

**Comment by martinstoeckli** [comment]
Is this requirement still active, are x86 devices still in use? I'm asking because I want to make the step to .Net-9 and supporting only x-64 would simplify the build procedure.
